diff --git a/CMake/HPHPFindLibs.cmake b/CMake/HPHPFindLibs.cmake
index 332c2f5..5b3e1ff 100644
--- a/CMake/HPHPFindLibs.cmake
+++ b/CMake/HPHPFindLibs.cmake
@@ -285,6 +285,18 @@ include_directories(${Mcrypt_INCLUDE_DIR})
 find_package(OpenSSL REQUIRED)
 include_directories(${OPENSSL_INCLUDE_DIR})
 
+# reSSL explicitly refuses to support RAND_egd()
+SET(CMAKE_REQUIRED_LIBRARIES ${OPENSSL_LIBRARIES})
+INCLUDE(CheckCXXSourceCompiles)
+CHECK_CXX_SOURCE_COMPILES("#include <openssl/rand.h>
+int main() {
+  return RAND_egd(\"/dev/null\");
+}" OPENSSL_HAVE_RAND_EGD)
+if (NOT OPENSSL_HAVE_RAND_EGD)
+  add_definitions("-DOPENSSL_NO_RAND_EGD")
+endif()
+
+
 # ZLIB
 find_package(ZLIB REQUIRED)
 include_directories(${ZLIB_INCLUDE_DIR})
diff --git a/hphp/runtime/ext/openssl/ext_openssl.cpp b/hphp/runtime/ext/openssl/ext_openssl.cpp
index 2227cd4..3b1e7a7 100644
--- a/hphp/runtime/ext/openssl/ext_openssl.cpp
+++ b/hphp/runtime/ext/openssl/ext_openssl.cpp
@@ -385,11 +385,13 @@ class php_x509_request {
     *seeded = 0;
     if (file == NULL) {
       file = RAND_file_name(buffer, sizeof(buffer));
+#ifndef OPENSSL_NO_RAND_EGD
     } else if (RAND_egd(file) > 0) {
       /* if the given filename is an EGD socket, don't
        * write anything back to it */
       *egdsocket = 1;
       return true;
+#endif
     }
 
     if (file == NULL || !RAND_load_file(file, -1)) {
